.vsts-ci.yml

```diff
@@ -55,6 +55,22 @@ phases:
             _BuildConfig: Release
             _PublishType: none
             _SignType: test
+  - template: /eng/build.yml
+    parameters:
+      agentOs: Windows_NT_TestAsTools
+      queue:
+        name: dotnet-external-temp
+        parallel: 2
+        timeoutInMinutes: 120
+        matrix:
+          Build_Debug:
```

Comment:
Do we really need debug and release for these?

Reply:
No, let's just do Debug